### PR TITLE
Some refactorings for the TimeAgo component

### PR DESCRIPTION
**Pull request by kulla**

### CODE REVIEW

File: src/components/time-ago.tsx [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-  opts?: timeago.Opts
-  className?: string
+interface TimeAgoProps extends Pick<ReactTimeAgoProps, 'datetime'> {
   dateAsTitle?: boolean
 }
 
-export function TimeAgo({
-  datetime,
-  opts,
-  className,
```
Comment (elbotho) [inline src/components/time-ago.tsx:17]: Please check if removing `className` breaks custom styled components (e.g. in notification.tsx we use a `<StyledTimeAgo/>`). This is the reason it was added.

**Review by elbotho — COMMENTED** [review]
see comment. rest looks a lot nicer, thanks!

File: src/components/time-ago.tsx (hunk 1 above)
Comment (kulla) [reply src/components/time-ago.tsx:17]: Is there a way to try the login feature locally?

Comment (elbotho) [reply src/components/time-ago.tsx:17]: sure, if you don't change the default setup, you should be able to login (via redirect to staging) using the normal login button.

Comment (kulla) [reply src/components/time-ago.tsx:17]: @elbotho Yes, you are right. Parameter `className` is needed. I have added it again as a paramater.

**Comment by kulla** [comment]
@elbotho Suggestions implemented. Ready for merge.

**Pull request merged**